app/pages/bookmarks.html

```diff
@@ -0,0 +1,33 @@
+<!--
+    Vieb - Vim Inspired Electron Browser
+    Copyright (C) 2019-2025 Jelmer van Arnhem
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="../colors/default.css" rel="stylesheet" type="text/css">
+    <title>Bookmarks</title>
+    <meta http-equiv="Content-Security-Policy" content="img-src file://*; object-src 'none'; style-src 'unsafe-inline' file://*; script-src 'none'; default-src 'none'">
+    <link rel="shortcut icon" type="image/png" href="../img/vieb.svg">
+</head>
+<body class="specialpage" id="bookmarkspage">
+    <div class="specialpage-controls">
+        <h1>Bookmarks</h1>
+        <button id="import-bookmarks-button" class="button">Import bookmarks</button>
```

Comment:
Class button is not needed when it is already a button element, nor is it a pattern that is used in our code, as there is no specific styling for the `.button` class, being a button element is enough.